ghost/admin/app/components/gh-nav-menu/footer.hbs

```diff
@@ -1,9 +1,16 @@
-<div class="gh-nav-bottom">
+<div class="gh-nav-bottom" {{did-insert this.loadCurrentMRR}}>
     {{#if this.hasThemeErrors}}
-    <button type="button" class="gh-theme-error-toast" {{on "click" (fn this.openThemeErrors null)}}>
-        <span class="gh-notification-title">Your theme contains errors</span>
-        <span class="gh-theme-error-p">Some functionality on your site may be limited &rarr;</span>
+    <button type="button" class="gh-footer-toast gh-theme-error-toast" {{on "click" (fn this.openThemeErrors null)}}>
+        <span class="gh-footer-toast-title gh-notification-title">Your theme contains errors</span>
+        <span class="gh-footer-toast-p">Some functionality on your site may be limited &rarr;</span>
     </button>
+    {{else if (and this.showReferralInvite (not this.hasThemeErrors))}}
+    <a href="https://referrals.ghost.org/?ref=admin" class="gh-referral-toast" target="_blank" rel="noopener noreferrer">
+        <span class="gh-referral-toast-close" {{on "click" (fn this.dismissReferralInvite)}}>&#xd7;</span>
+        <strong>You qualify for our invite-only  referral program.</strong>
+        <p class="gh-footer-toast-p">Refer people to Ghost and earn a <strong>30% share</strong> of the subscription revenue, every single month.</p>
+        <button class="gh-btn gh-btn-black gh-referral-toast-button"><span>Find out more &rarr;</span></button>
+    </a>
```

Comment:
It would be cleaner to pull this into a separate component, then you can write it as a glimmer component and use it's `constructor` to load the MRR stats rather than `{{did-insert}}` which we generally want to avoid (it's designed as a band-aid whilst migrating from old-style to new-style Ember and is [generally considered an anti-pattern](https://github.com/emberjs/ember-render-modifiers#when-to-use-these-modifiers-and-when-not-to-use-them)).

Means there's less old-style code that needs migrating later down the line and the show/not-show/dismiss logic is all self-contained rather than being mixed in with the other nav footer code 😄 (+ easier to test, easier to move when designs change)